README.md

```diff
@@ -1,128 +1,201 @@
+```python
+    from matplotlib_visual.piechart import display
+    display("<filename-without-extension>")
 ```
 > *`piechart` method writes the file in the root directory of execution.*  
 
-- **Web Scraping** from a cl1p.net clipboard. 🌐  
-```python  
-from package.display import grab  
-grab('url-name')  
+- **📍Web Scraping** from a cl1p.net clipboard.
+```python
+     from package_name.clp import grab
+     grab('url-name')
 ```
 > *`grab` method retrieves the clipboard's text content from the specified URL and displays it in the system console. If no content is found, a message indicating "nothing found" is returned.*  
 
 ---
 
-## Contribution Guidelines 🙌
+5. **✅ Finalize Your Changes**
+   - Before you proceed to next step ensure all changes are made and check them once again before making the final commit.
 
-We welcome all contributions! Whether you're improving features, fixing bugs, or enhancing documentation, your help is valuable to us. Here's how to get started:
 
-### 1. Understand the Project 🔍
-- It’s **beginner-friendly** with most features implemented in **minimal lines of code**.  
-- Research thoroughly to understand how to implement stealthy features. 🕵️‍♀️
+6. **💬 Commit Your Changes** 
+   - Once ready, commit them with a descriptive message:
+     ```bash
+     git add .
+     git commit -m "Added feature X or Fixed issue Y"
+     ```
 
-### 2. Find or Create an Issue 🛠️
-- Visit the [**Issues section**](https://github.com/AmateursLeague/sneaky-package/issues) in this repository.  
-- Look for existing issues that are available to work on or past their time limit! ⏳  
-- Or raise your own issue with a proposal or idea and start contributing! 🗣️
+7. **📤 Push Your Changes**
+   - Push your changes to your forked repository:
+     ```bash
+     git push origin YOUR-BRANCH-NAME
+     ```
 
-### 3. Make Improvements ✨
-- Keep the code **clean, simple, and efficient**. 🧹
+8. **🔄 Create a Pull Request (PR)** 
+   - Go back to the original repository [here]((https://github.com/AmateursLeague/sneaky-package)).
+   - Click the *Compare & pull request* button, write a short description of your changes, and submit the PR.
 
-### 4. Show Your Support 🌟
-- If you find this project useful or interesting, please **star the repository** on GitHub! ⭐ It helps the project grow!
+9. **🔎 Review Changes**
+   - The project manager will review your PR, and if approved, your request will be merged.
 
----
 
-## Key Points to Remember 📌
-- This package integrates source files into a target machine using the `stash/` directory. 📁  
-- A **test file** is provided—run it to ensure that the files inside the stash appear on the target machine. 💡  
-- Filenames like `graph.py`, `models.py`, and `piechart.py` are **aliases** to avoid detection. 🔒  
+## 🏆 Contribution Points
+All tasks will be assigned various levels based on complexity and required skills. Each level provides different points:
```

Comment:
This is GSSOC-EXT specific, a project readme must focus on the project itself, not on other things. Youcani cludea link for more info about the program elsewhere.